src/codecoverage/bot/tests/test_artifacts.py

```diff
@@ -98,8 +98,8 @@ def _group_tasks():
             ('test-linux64-ccov/debug-mochitest-devtools-chrome-e10s-4', 'completed'),
         ],
         [
-            ('test-linux64-ccov/debug-xpcshell-4', 'exception'),
-            ('test-linux64-ccov/debug-xpcshell-4', 'failed'),
+            ('test-windows10-64-ccov/debug-xpcshell-4', 'exception'),
+            ('test-windows10-64-ccov/debug-xpcshell-4', 'failed'),
```

Comment:
Why change the platform here?

Reply:
I wanted to make sure in `test_download_all` we had at least one Win and one Linux, to make sure there's nothing in the code wrongly filtering by platform.
Since I removed win talos, this was no longer true.